public/schemas/employment.json.erb

```diff
@@ -37,20 +37,16 @@
                 },
                 "national_insurance": {
                   "$ref": "<%= "file://#{@schema_dir}/common.json#currency" %>"
-                },
-                "net_employment_income": {
-                  "$ref": "<%= "file://#{@schema_dir}/common.json#positive_currency" %>"
```

Comment:
The only thing is that by removing this field, CFE goes from being responsible for checking (admittedly implicitly) that `gross income > tax + NI` and telling the client if there's a problem, to not doing that check. In an ideal world maybe CFE would still validate that  net employment income is positive, it would just calculate net employment income itself as part of it. But I get that in practice it's not necessary given CCQ and Apply already do those checks, so maybe at best it's a separate future feature request.

Reply:
@patrick-laa I wonder if that's why this field was here? IMHO we need to revisit the JSON schema validation for just this sort of reason (the technology doesn't allow for all the validations we would like to do - like this one)

Reply:
@patrick-laa have added a rails validation so that this is now reported back to the client